src/components/community/community-base.js

```diff
@@ -19,4 +18,9 @@ export class CommunityBase extends LitElement {
 		super.connectedCallback();
 		this.communityArticleDirective = communityUrlFactory(this.articleMap);
 	}
-}
+
+	update() {
+		super.update();
+		this.communityArticleDirective = communityUrlFactory(this.articleMap);
+	}
```

Comment:
Should be `updated` (not `update`) and this should get called always for new values as well so you don't need to set it in the constructor.

```suggestion 
	updated(changedProperties) {
		if (changedProperties.has('articleMap')) {
			this.communityArticleDirective = communityUrlFactory(this.articleMap);
		}
	}
```